sdk/confidentialledger/confidential-ledger-rest/test/public/usersEndpoint.spec.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { ConfidentialLedgerClient, GetUser200Response } from "../../src";
+import { Recorder, env } from "@azure-tools/test-recorder";
+import { createClient, createRecorder } from "./utils/recordedClient";
+
+import { Context } from "mocha";
+import { assert } from "chai";
+
+describe("Get user", () => {
+  let recorder: Recorder;
+  let client: ConfidentialLedgerClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = createRecorder(this);
+    client = await createClient();
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  // TODO: add subLedgerIds test
```

Comment:
Can you please file an issue to track this work and link it here?

Reply:
I added the test somewhere else; my bad!